### PR TITLE
add missing int()

### LINKED ISSUE #81: Typo in high scores
https://github.com/ThePhilgrim/minesweeper/blob/bf0a5358445b22a92f267a419cb17505ab12a065/minesweeper.py#L343

Should be `int(seconds / 60)`. Introduced in #79

### PR DESCRIPTION
**Pull request by Akuli**
Fixes #81

### CODE REVIEW

**Review by ThePhilgrim — COMMENTED** [review]
I'll leave it up to Arrinao to approve or request changes on this, since you guys wrote it.

It 100% okay that you guys change and work on code that I've written, but please talk to me about it at least so I learn how I can improve my coding. If I write it and somebody changes it without saying anything, I don't learn anything

**Comment by Akuli** [comment]
This code wasn't changed because it was bad. It was changed because the type of `highscore_dict["time"]` changed from `int` to `float`, and because your code expects it to be an `int`, we added `round()`.

**Comment by ThePhilgrim** [comment]
> This code wasn't changed because it was bad. It was changed because the type of `highscore_dict["time"]` changed from `int` to `float`, and because your code expects it to be an `int`, we added `round()`.

Ah okay. I thought you were only doing improvements in the script. I'd just like to know how I can optimize the efficiency of my code, that's why.

File: minesweeper.py [hunk 1]
```diff
@@ -340,7 +340,7 @@ def get_avg_time(highscore_dict):
     for highscore_dict in sorted(json_dict["high_scores"], key=get_highscore_data):
         seconds = round(highscore_dict["time"])
         if seconds >= 60:
-            format_time = f"{seconds / 60} min & {seconds % 60} sec"
+            format_time = f"{int(seconds / 60)} min & {seconds % 60} sec"
```
Comment (ThePhilgrim) [inline minesweeper.py:343]: So in this sense, you're just adding back "int" which was in my code previously.  87 seconds without int would become "1.45 min & 27 sec" whereas with "int" it becomes "1 min & 27 sec"

**Review by ThePhilgrim — APPROVED** [review]

File: minesweeper.py (hunk 1 above)
Comment (Akuli) [reply minesweeper.py:343]: Right. I'm just adding an `int()` that was accidentally deleted.

**Pull request merged**